pkg/client/client.go

```diff
@@ -366,6 +372,7 @@ func (c *client) startActions() {
 			actionsCtx, actionsCancel := context.WithCancel(c.ctx)
```

Comment:
same here, it should say `defer actionsCancel()`. The function body/loop requires some refactoring to ensure proper cleanup no matter what.

Reply:
fixed